Dockerfile

```diff
@@ -1,11 +1,24 @@
-FROM ubuntu:22.04
+FROM ubuntu:jammy
 ARG VERSION
 
-RUN apt update && apt install -y \
+ENV TZ=UTC
+
+RUN set -eux; \
+# install python
+    apt-get update; \
+    DEBIAN_FRONTEND=noninteractive apt-get full-upgrade -y; \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
     python3.10 \
-    python3-pip && \
-    pip install --no-cache mlflow==$VERSION && \
-    apt autoremove -yqq --purge && \
-    apt clean
+    python3-pip \
+    tzdata \
+    ; \
+    DEBIAN_FRONTEND=noninteractive apt-get remove --purge --auto-remove -y; \
+    rm -rf /var/lib/apt/lists/*; \
+# install mlflow
+    pip install --no-cache mlflow==$VERSION; \
+# create manifest for security scan
+    mkdir -p /usr/share/rocks; \
```

Comment:
```suggestion
    rm -rf /var/lib/apt/lists/*; \
    # install mlflow
    pip install --no-cache mlflow==$VERSION; \
    # create manifest for security scan
    mkdir -p /usr/share/rocks; \
```

Reply:
for me that looks weird, if you write python, you probably will never do like this

Reply:
I dont know I think to stay consistent I will leave it as this

Reply:
I don't think you should focused too much on how others are doing things :) cause maybe you'll have the chance to do them better!

Anyway, I'd also indent the comments, but that's a nitpick. as you prefer

Reply:
btw, one suggestion I would make as well, is to move this metadata file creation into a separate layer. Why? Well, you seemed a bit concerned about diverging this Dockerfile from the upstream one. If you have 2 layers you'll at least keep the 1st one aligned with upstream. Just a design choice...should harm the end result. It's up to you